frontend/src/components/Simulator/ContractMethodItem.vue

```diff
@@ -79,7 +79,7 @@ const resetInputs = () => {
     switch (input.type) {
       case 'uint256':
       case 'float':
-        defaultValue = 0;
+        defaultValue = '';
```

Comment:
This seems weird. Why this change?

Reply:
Basically I wanted to allow passing an empty value to be able to test optional parameter. Does that make sense?

Reply:
I actually think this change was not necessary, reverting